index.html

```diff
@@ -38,5 +38,161 @@ <h2>
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="about-me">
+              <h1>About me</h1>
+              <p>
+                  Hello I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and experience! 
+                  If you like what you see and have a project you need coded, 
+                  don’t hestiate to contact me.
+              </p>
+              <button>Get My Resume</button>
+              <hr>
+            </div>
+            <div class="frames">
+              <div class="frame">
+                <div class="frame1">
+                  <img src="images/Rectangle 55.png" alt="">
+                  <h1>Languages</h1>
+                </div>
+                <div class="languages">
+                  <h2 class="javascript">Javascript</h2>
+                  <h3 class="ruby">Ruby</h3>
+                  <h3 class="html">Html</h3>
+                  <h3 class="css">Css</h3>
+                </div>
+              </div>
+              <div class="frame">
+                <div class="frame1">
+                  <img class="frameworklogo" src="images/Rectangle 56.png" alt="">
+                  <h1 class="framework">FrameWorks</h1>
+                </div>
+                <div class="languages">
+                  <h3 class="boostrap">Boostrap</h3>
+                  <h3 class="rails">Ruby on Rails</h3>
+                  <h3 class="respec">RsPec</h3>
+                  <h3 class="capybara">CapyBara</h3>
+                  <h3 class="sele">Selenium</h3>
+                </div>
+              </div>
+              <div class="frame">
+                <div class="frame1">
+                  <img class="frame3logo" src="images/Rectangle 57.png" alt="">
+                  <h1 class="skill">Skills</h1>
+                </div>
+                <div class="languages">
+                  <h3 class="boostrap">CodeKit</h3>
+                  <h3 class="github">Github</h3>
+                  <h3 class="respec">Codepen</h3>
+                  <h3 class="gitlab">Gitlab</h3>
+                  <h3 class="terminal">Terminal</h3>
```

Comment:
[OPTIONAL]
- [ ] For semantic reasons, the tech tasks are not supposed to be inside the `header tag` as is the case here in your code. You may consider grabbing those inside an `unordered list ul`